processing/src/main/java/io/druid/query/topn/TopNMapFn.java

```diff
@@ -33,6 +33,10 @@
 
 public class TopNMapFn
 {
+  public static final long ZERO_LONG = 0L;
```

Comment:
If this is to avoid boxing, type should be `Long`, not `long`, otherwise it will still be boxing. Also suggested to pull this to some global place, like druid common "Primitives".